SmartDeviceLink/NSMutableDictionary+Store.m

```diff
@@ -35,9 +40,7 @@ - (nullable id)sdl_objectForName:(SDLName)name ofClass:(Class)classType {
 
 - (NSArray *)sdl_objectsForName:(SDLName)name ofClass:(Class)classType {
     NSArray *array = [self sdl_objectForName:name];
-    if ([array isEqual:[NSNull null]]) {
```

Comment:
Isn't this a breaking change? If a developer set any array value to `[NSNull null]` then they will expect to get an empty array back instead of `null`.

Reply:
I don't think so. setting a parameter to `nil` or `NSNull` should be no difference. When I want to set an empty array developers should use `@[]` or mutable equivalents.